plugins/auth-backend/src/identity/TokenFactory.ts

```diff
@@ -140,8 +153,18 @@ export class TokenFactory implements TokenIssuer {
       });
     }
 
-    // NOTE: we're currently only storing public keys, but if we start storing private keys we'd have to convert here
-    return { keys: validKeys.map(({ key }) => key) };
+    // NOTE: we're currently only storing public keys, but if we start storing private keys we'd have to convert here.
+    // NOTE: If allowGuestMode flag is set to true add a static EC256 guest key to the list of valid public keys that the identity client can authenticate against. This key will only be used in "development" and "test" processes. Key will not be set as valid for "production".
+    return {
+      keys: validKeys
+        .map(({ key }) => key)
+        .concat(
```

Comment:
much easier to read if you do an `if (...) { validKeys.push(...) }` above

Reply:
Changed per suggestion to an if conditional. Thanks.